Facebooky/FormSohrtcutsSettings.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Specialized;
+using System.Windows.Forms;
+
+namespace Facebooky
+{
+	public partial class FormShortcutsSettings : Form
+	{
+		private readonly StringDictionary r_ShortcutsDickDictionary;
+
+		public FormShortcutsSettings(StringDictionary i_ShortcutsDickDictionary)
+		{
+			r_ShortcutsDickDictionary = i_ShortcutsDickDictionary;
+			InitializeComponent();
+			foreach (string key in r_ShortcutsDickDictionary.Keys)
```

Comment:
Originally I did it this way but got this exception:
Complex DataBinding accepts as a data source either an IList or an IListSource.
So I changed the code to what it is now.